package.json

```diff
@@ -28,11 +28,13 @@
     "@scure/bip32": "^1.4.0",
     "@tanstack/react-query": "^5.28.14",
     "@tanstack/react-query-next-experimental": "^5.28.14",
+    "@uidotdev/usehooks": "^2.4.1",
     "axios": "^1.6.8",
     "bitcoinjs-lib": "^6.1.5",
     "btc-staking-ts": "^0.2.8",
     "date-fns": "^3.6.0",
     "framer-motion": "^11.1.9",
+    "lodash.debounce": "^4.0.8",
```

Comment:
Looks like we shifted from `lodash.debounce` to `@uidotdev/usehooks` `useDebounce`?
If we don't use `lodash` we need to remove it